target/linux/qualcommax/image/ipq60xx.mk

```diff
@@ -23,3 +23,19 @@ define Device/netgear_wax214
        DEVICE_PACKAGES := ipq-wifi-netgear_wax214
 endef
 TARGET_DEVICES += netgear_wax214
+
+define Device/linksys_mr7350
+	$(call Device/FitImage)
+	DEVICE_VENDOR := Linksys
+	DEVICE_MODEL := MR7350
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	KERNEL_SIZE := 8192k
+	IMAGE_SIZE := 75776k
+	NAND_SIZE := 256m
+	SOC := ipq6018
+	IMAGES := factory.bin
+	IMAGE/factory.bin  := append-kernel | pad-to $$$$(KERNEL_SIZE) | append-ubi | linksys-image type=MR7350
+	DEVICE_PACKAGES := kmod-leds-pca963x ipq-wifi-linksys_mr7350 kmod-usb-dwc3-qcom
```

Comment:
`kmod-usb-dwc3-qcom` is included by default: https://github.com/openwrt/openwrt/blob/main/target/linux/qualcommax/Makefile#L15

Reply:
Done